src/webapi/document.rs

```diff
@@ -116,4 +116,17 @@ impl Document {
             js!( @(no_return) @{self}.title = @{title}; );
         }
     }
+
+    /// Returns the Element that is the root element of the document (for example, the <html>
```

Comment:
The `<html>` has to be escaped in backticks or it will mess up the docs.

Reply:
this should be good now, I've also gone back and added backticks that were missed in some of my previous other work.

Reply:
also updated doc links and the change to `document_element()` (:
lmk if you spot anything else!